client/post-editor/editor-confirmation-sidebar/index.jsx

```diff
@@ -46,9 +46,9 @@ class EditorConfirmationSidebar extends React.Component {
 			case 'update':
 				return this.props.translate( 'Update' );
 			case 'schedule':
-				return this.props.translate( 'Schedule' );
+				return this.props.translate( 'Schedule!' );
 			case 'publish':
-				return this.props.translate( 'Publish' );
+				return this.props.translate( 'Publish!' );
```

Comment:
:disappointed: 104 existing translations will be lost with this change.
If the string must change, the following option has already been translated **51** times:
`translate( 'Publish', { context: 'admin bar menu group label'} )` *ES Score: 13*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/editor-confirmation-sidebar-label)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).